ansible/playbooks/roles/redpanda_broker/vars/main.yml

```diff
@@ -3,4 +3,6 @@
 custom_config_templates:
   - template: configs/defaults.j2
   - template: configs/tls.j2
-    condition: "{{ tls | default(False) | bool }}"
\ No newline at end of file
+    condition: "{{ tls | default(False) | bool }}"
+  - template: configs/tiered_storage.j2
+    condition: "{{ tiered_storage_bucket_name is defined | default(False) | bool }}"
```

Comment:
The `tiered_storage_bucket_name` ansible variable is pulled from `hosts.ini`, and is only defined if the `tiered_storage_enabled` terraform variable is `true`.